packages/integration-karma/test/template/attribute-hidden/index.spec.js

```diff
@@ -0,0 +1,56 @@
+import { createElement } from 'lwc';
```

Comment:
We don't provide special handling the `hidden` attribute with this updated version of the PR. Can we rename this test to `attributes-boolean-global-html`?